rust/template/cmd_parser/parse.rs

```diff
@@ -65,7 +65,10 @@ named!(pub parse_command<&[u8], Command>,
                             apply!(sym,";")         >>
                             (Command::Commit(delta.is_some())))                                 |
                   do_parse!(apply!(sym,"timestamp") >> apply!(sym,";") >> (Command::Timestamp)) |
-                  do_parse!(apply!(sym,"#")         >> take_until!("\n") >> (Command::Comment)) |
+                  do_parse!(apply!(sym,"#")         >>
+                            take_until!("\n")       >>
+                            apply!(sym,"\n")        >>
```

Comment:
Will this work if the comment is on the last line of the command file?

Reply:
Fair point. Based on my experimentation that is not a problem. I had a comment on the last line (and a bunch of actual commands before) and then tested with and without a trailing newline symbol. Both cases behaved identically.